src/connection.js

```diff
@@ -1126,6 +1158,25 @@ const ConfirmedTransactionMetaResult = struct.union([
   struct.pick({
     err: TransactionErrorResult,
     fee: 'number',
+    innerInstructions: struct.array([
```

Comment:
We'll need both a `ConfirmedTransactionMetaResult` and a `ParsedConfirmedTransactionMetaResult`. Can you please add `ParsedConfirmedTransactionMetaResult` and look at `ConfirmedTransactionResult` and `ParsedConfirmedTransactionResult` for the difference between how instructions are shaped in those validator structs?

Reply:
Also, since `innerInstructions` is a relatively new feature, a developer could use a local cluster which doesn't return this information so we need to use `struct.union(['undefined', ...])`. And also `innerInstructions` can be `null` for older transactions that were processed before we started tracking inner instructions so you'll want to use `struct.union(['undefined', 'null', ...])`